game_compatibility.md

```diff
@@ -0,0 +1,21 @@
+✅ - reported as working
+❓ - not reported or not possible to check (lack of HDR display etc)
+❌ - reported not working
+
+| Game                        | Game Version | Mod version | Mod install | FrameGen Works | Ingame HDR | Other issues                                           |
+|-----------------------------|---------|-------------|-------------|-------|------------|----------------------------------------------------------------------|
+| Cyberpunk 2077              | 2.1     | 0.5         | DLSSTweaks ini method  | ✅     | ✅          | Ghosting/artifact line at the bottem of the screen when driving fast, reported no ghosting when using no upscale/FSR2 upscale instead of DLSS, can be issue of DLSS upscale or FG bug |
+| The Witcher 3 NextGen       | 4.04    | 0.5         | DLSSTweaks ini method  | ✅     | ✅          | Looks like no issues                                   |
+| Ratchet & Clank: Rift Apart | v1.922  | 0.5         | DLSSTweaks ini method  | ✅     | ✅          | Garbled UI even without any upscale enabled            |
+| Alan Wake 2                 | 1.0.13  | 0.5         | nvngx.dll method       | ✅     | ✅          | Ghosting around player character in DLSS upscale mode  |
+| Spider-Man: Miles Morales   | v1.1130.0.0  | 0.5    | DLSSTweaks ini method  | ✅     | ❓ | Looks like no issues                          |
+| Hogwarts Legacy             | ❓(newest steam)  | 0.41, 0.5  | nvngx.dll method  | ✅ | ❓ | No UI artifacts, can not use DLSS Sharpening with FG, causes game crash  |
```

Comment:
UPDATE HOGWARTS LEGACY: After some time playing the game the nvidia driver crash without reporting any errors inside the windows log (nvlddmkm), this seems to be a common issue from others that were able to run FG on this game (both nvngx.dll and DLSSTweaks method). THIS DOES NOT HAPPEN WITHOUT THE MOD.

Reply:
Thanks, added info to the list.